mos-platform/atari8-mega/link.ld

```diff
@@ -0,0 +1,372 @@
+/* Atari800 MegaCart and SIC! (bank-switched) cartridge linker script.
+
+  Support for 16 KiB to 512 KiB MegaCart and SIC! cartridges.
+
+  There are be "__cart_rom_size / 16 KiB" banks, mapped one at a time
+  over the entire 16 KiB cartridge address space from 0x8000 through
+  0xBFFF.  It is also possible to disable the cartridge to access
+  underlying RAM.
+
+  On power-up the bank0 is selected, however, SIC! cartridges-up will
+  not have the lower 8 KiB enabled.  We use INIT to fix this and
+  adjust MEMTOP if needed.
+
+  https://github.com/atari800/atari800/blob/ATARI800_5_0_0/DOC/cart.txt
+  has details. */
+
+INPUT(tail0.o)
+
+/* Default cartridge size.  This isn't the largest possible size to
```

Comment:
Instead of defaulting to a non-extremum size, adding test for successful compilation would be a more principled solution. Is there another reason to justify 128 KiB specifically as the default size?

Reply:
Can do. PTAL.